scene/gui/spin_box.cpp

```diff
@@ -397,13 +547,50 @@ void SpinBox::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "custom_arrow_step", PROPERTY_HINT_RANGE, "0,10000,0.0001,or_greater"), "set_custom_arrow_step", "get_custom_arrow_step");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "select_all_on_focus"), "set_select_all_on_focus", "is_select_all_on_focus");
 
+	BIND_THEME_ITEM_CUSTOM(Theme::DATA_TYPE_CONSTANT, SpinBox, buttons_v_separation, "buttons_vertical_separation");
```

Comment:
That looks like overusing `BIND_THEME_ITEM_CUSTOM`. It should only be used when cache entry does not match the item name and, unless it improves clarity, they should always match.

Reply:
I still have to understand those binding macros better. I favor human readability in displayed property names in the editor, is having just "v" acceptable instead of "vertical"?

Reply:
Need to get more familiar with those BIND_THEME_ macros.

Is it acceptable if the user sees "v" instead of "vertical"? I put the priority in human readability initially.